### PR TITLE
NiftyMIC docker path

### PR DESCRIPTION
**Pull request by t-sanchez**
- Add a docker config, and compatibility to the NiftyMIC docker. 
- Fix the typo niftimic in the code
- Work on the documentation

**ToDo**
Test that the singularity path is not broken.

### CODE REVIEW

**Comment by davidmeunier79** [comment]
@t-sanchez is it good to merged? or do you still have modifications and checks to do?

**Comment by t-sanchez** [comment]
This should be good to be merged. To be sure however that it works on all of our settings, could you or @alexpron test it prior to merging?

**Comment by davidmeunier79** [comment]
> This should be good to be merged. To be sure however that it works on all of our settings, could you or @alexpron test it prior to merging?
> 
> Especially given that I cannot test it on singularity.

OK I will give it a try if get back to you

File: workflows/params_segment_fet_niftimic.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
     "general":
     {
         "pre_command": "singularity exec -B /scratch:/scratch ",
-        "niftimic_image": "/scratch/apron/softs/niftymic.multifact_latest.sif"
+        "niftymic_image": "/scratch/apron/softs/niftymic.multifact_latest.sif"
```
Comment (davidmeunier79) [inline workflows/params_segment_fet_niftimic.json:5]:
```suggestion
        "niftymic_image": " /scratch/apron/softs/niftymic.multifact_latest.sif "
```

**Pull request merged**